manifests/controller/config.pp

```diff
@@ -5,17 +5,32 @@
 # Addtional details on the options can be found at
 # https://puppet.com/docs/bolt/latest/bolt_configuration_options.html.
 #
+# @param local_user
+#   The local account to be used for running Bolt. The default is the $username account specified
+#   in the user.pp manifest.
+#
 # @param local_group
 #   The local group to be used for file permissions associated with the local_user account. The
 #   default is the $username account specified in the user.pp manifest.
 #
-# @param local_user_home
+# @param local_home
 #   The home directory of the local account to be used for running Bolt. The default is the
 #   $home directory for the account specified in the user.pp manifest.
 #
-# @param local_user
-#   The local account to be used for running Bolt. The default is the $username account specified
-#   in the user.pp manifest.
+# @param simp_omni_environment
+#   Use the SIMP Omni-Environment.
+#
+# @param simp_environment_name
+#   The name of the SIMP Omni-Environment to use.
+#
+# @param puppet_env_path
+#   The path to the Puppet environment in the SIMP Omni-Environment.
+#
+# @param secondary_env_path
```

Comment:
In general, this isn't going to work unless you are `root` and you really shouldn't be running Bolt as `root`.

Reply:
I think this allows the flexibility for a user to create SIMP Omni-environment whereever they would like, for example, in their home directory, in a shared location in the /var directory if there are multiple bolt users, or transition from an existing puppet setup to bolt.

An example is provided in the simp_bolt acceptance test in https://github.com/m-morrone/pupmod-simp-simp_bolt/tree/SIMP-7124 where the vagrant user creates a SIMP Omni-environment in its home directory.